### PR TITLE
Add Smoke Effect on Enemy Spawn From Relic

### LINKED ISSUE #74: Add smoke effect to enemies that appear on dome hit

### PR DESCRIPTION
**Pull request by daniela-venuta**
Closes #74

Adds smoke effect and a short delay before enemies spawn to give the player a warning

### CODE REVIEW

File: Hush/Assets/Scripts/Relics/RelicEnemySpawner.cs [hunk 1]
```diff
@@ -18,14 +20,27 @@ void SpawnEnemies()
...
+      private void CreateSmoke(Vector3 position)
+      {
+         var smokeClone = Instantiate(smokePrefab, position, Quaternion.identity);
+         smokeClone.GetComponent<ParticleSystem>().Play();
+         Destroy(smokeClone, 1.5f);
+      }
+
+      IEnumerator CreateEnemy(Vector3 position)
+      {
+         yield return new WaitForSeconds(.3f);
```
Comment (po-trottier) [inline Hush/Assets/Scripts/Relics/RelicEnemySpawner.cs:39]: delay should be SerializeField

File: Hush/Assets/Scripts/Relics/RelicEnemySpawner.cs [hunk 2]
```diff
@@ -18,14 +20,27 @@ void SpawnEnemies()
...
+         
+         var enemyClone = Instantiate(enemyPrefab, position, Quaternion.identity);
+         var target = GameObject.FindWithTag(Tags.Relic);
```
Comment (po-trottier) [inline Hush/Assets/Scripts/Relics/RelicEnemySpawner.cs:42]: Don't need to find the target every time you spawn an enemy... should either be a serializeField or Find 1x and then set this

**Review by po-trottier — CHANGES_REQUESTED** [review]
2 minor changes

**Review by emilie-martin — APPROVED** [review]
navmesh is broken but that's gonna be taken care of in another PR (#112 ) so you're good to go for me; it looks good! ☁️

**Review by po-trottier — APPROVED** [review]
image

**Review by arejayelle — APPROVED** [review]
works fine.

### minor
The smoke effect would look better if it was much shorter (since the enemies are popping into the scene instantly) and possibly make it circular like it's radiating on the floor off of them.  

Something like this: 
https://user-images.githubusercontent.com/35614138/143910210-f3bf7acf-de82-4663-811a-aa9e3e0a7db1.mp4

**Comment by daniela-venuta** [comment]
> works fine.
> 
> ### minor
> The smoke effect would look better if it was much shorter (since the enemies are popping into the scene instantly) and possibly make it circular like it's radiating on the floor off of them.
> 
> Something like this: https://user-images.githubusercontent.com/35614138/143910210-f3bf7acf-de82-4663-811a-aa9e3e0a7db1.mp4

I'm gonna merge this for now as the duration is a serialized field so we can change this in the editor pretty easily. Can make more circular later

**Pull request merged**